tests/run-macros/tasty-getfile-implicit-fun-context/Macro_1.scala

```diff
@@ -11,7 +11,7 @@ object SourceFiles {
 
   def getThisFileImpl: Macro[String] = {
     val qctx = tastyContext
-    import qctx.tasty.{_, given _}
+    import qctx.tasty._
```

Comment:
We have many such cases in the test set, change them too?

Reply:
We still need to adapt unseal to be able to change them all

Reply:
See #8441